### ISSUE TITLE
Known vulnerability found notification

### ISSUE THREAD

**Issue by leac** [JosePFs/moodle-filter_tabs#8]
Hi

I get a "Known vulnerability found" notification on Github regarding this filter. Is there a way to fix this?

Thank you

**Comment by abias** [comment]
Hi Lea,

I am the former maintainer of this Moodle plugin, but I have handed it over to @JosePFs some time ago and don't use it in production anymore.

I would only be curious to understand where in Github you are seeing this message. Can you please add a URL or screenshot?

Thanks,
Alex

**Comment by leac** [comment]
Hi Alex

Thank you for your quick response. 
I cloned this repository into our Moodle instance, and am maintaining it on Github, and now I get this message:
![image](https://user-images.githubusercontent.com/1826720/44205496-423c1900-a15f-11e8-8be1-48a2b6ef3949.png)

I think this has to do with the filter, and that's why I'm asking this here. I hope I'm not terribly mistaken...

**Comment by abias** [comment]
Hi Lea,

thanks for the explanation and the screenshot.

package-lock.json is a file which was added by @JosePFs after I have handed the repository over to him, so I don't know exactly why he has added this file.

However, I think I can tell you that this file is used for installing development tools into the Moodle plugin and won't be processed if you run the plugin in production. I think @JosePFs can solve these vulnerable dependencies quite easily.

In the meantime, you would be safe to delete the package.json, package-lock.json and Grundfile.js files in your fork to be really sure.

Cheers,
Alex
